waku/waku.md

```diff
@@ -495,11 +495,9 @@ Notes useful for implementing Waku mode.
 
 ### Node discovery
 
-[Discovery v4](https://github.com/ethereum/devp2p/wiki/Discovery-Overview) SHOULD NOT be used, because it doesn't distinguish between capabilities. It will thus have a hard time finding Waku/Whisper nodes.
+[Discovery v5](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md) MAY be used. However, it is quite bandwidth heavy for resource restricted devices.
```

Comment:
Which draft version?

Reply:
wdym?

Reply:
Discovery v5 isn't finalized, so it would probably be a good idea to specify which version we are targeting

E.g. Draft of October 2019. (https://github.com/ethereum/devp2p/commit/193bc09c807936bbcb7b484d9bdbd780f463e3bd) (can be in a footnote if is too wordy)

Reply:
ah fair, I am not entirely sure of which version I think @adambabik or @cammellos can answer that.

Reply:
We use the deprecated version. It has been [officially deprecated](https://github.com/ethereum/go-ethereum/pull/20367) and it's not compliant with the [current discv5 spec](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md). 

I would only keep info about EIP-1459 here.

Reply:
Ok, I could remove the entire discv5 stuff, @oskarth how do you feel?

Reply:
Since this is the Waku spec and not Status v1 reality, sure